app/index.html

```diff
@@ -7,6 +7,12 @@
 </head>
 <body class="container">
 <center><h1>Oasis Secret Ballot dApp</h1></center>
+<center>
+    <div id="warning-div">
+        <p>Reminder: Please ensure you are using the Oasis Devnet (configured on MetaMask) and received sufficient DEV from 
+            the Oasis Faucet. <br><a href="https://docs.oasiscloud.io/en/latest/secret-ballot/">See tutorial for details</a>.</p>
```

Comment:
```suggestion
            the Oasis faucet. <br><a href="https://docs.oasiscloud.io/en/latest/secret-ballot/">See tutorial for details</a>.</p>
```

I think we haven't been capitalizing "faucet" in our other copy